src/pages/waiting-room-to-car/components/instructor-registration/instructor-registration.html

```diff
@@ -0,0 +1,15 @@
+            <ion-row class="mes-component-row mes-row-separator" id="instructor-registration-card" [formGroup]="formGroup">
+                <ion-col class="component-label" col-32 align-self-center>
+                  <label>Instructor registration number (optional)</label>
+                </ion-col>
+                <ion-col align-self-center>
+                  <ion-row>
+                    <ion-col col-32>
+                      <input #instructorRegistrationInput type="number" id="instructor-registration" numbersOnly
```

Comment:
This template variable is not needed anymore.

Reply:
removed